### ISSUE TITLE
Add Packard Bell Phoenix

### ISSUE THREAD

**Issue by TheDell1998** [86Box/86Box#3190]
So you're think you're adding the Packard Bell Phoenix to your 86Box?

**Comment by OBattler** [comment]
Closed for being too vague.

**Comment by DDXofficial** [comment]
Closed issue, but regardless:

You've requested several Packard Bell machines over the course of these past few weeks and the developers have stated numerous times in multiple issue threads that implementing those machines will either be difficult or take a while.

Please do NOT waste your time requesting such machines again.

**Comment by pankoza-pl** [comment]
> Closed issue, but regardless:
> 
> You've requested several Packard Bell machines over the course of these past few weeks and the developers have stated numerous times in multiple issue threads that implementing those machines will either be difficult or take a while.
> 
> Please do NOT waste your time requesting such machines again.

I agree, and most of the machines that TheDell1998 are in fact impossible to add, but the Packard Bell Phoenix in particular should be easy to add, because it's just a MSI MS-6199VA, which is a pretty standard Via Apollo Pro 133 Slot 1 board, with a onboard AudioPCI audio that we emulate
